### DISCUSSION TITLE
Enrage Weakaura not Correctly Reskinning

### DISCUSSION THREAD

**Discussion by parasemic** [SFX-WoW/Masque#179] [Issues] [answered]
To be honest, I'm not certain whether this is a bug within Masque or WeakAuras but each of my auras is working correctly, except an "Enrage" buff timer on my Warrior. What happens is when the WA is created (or duplicated) the skin is applied but after a reload or any loading screen, the skin disappears and never returns. 

I've tried creating the WA from both Aura trigger and from combat log event. I've also tried manually changing the icon to a different one, but that doesn't help either.

E: Changing the icon to one that isnt either "Enrage Hatchling" (136224) or "Enrage" (136215) seems to have solved the issue.

**Comment by StormFX** [comment, marked as answer]
This doesn't seem to be a Masque issue.
